pkgs/data/icons/antu-icon-theme/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitLab, gtk3 }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "antu-icon-theme";
+  version = "ff16444b";
+
+  src = fetchFromGitLab {
+    owner = "froodo_alexis";
+    repo = "antu-icons";
+    rev = version;
+    sha256 = "sha256-CLcr+X/b0moVEBV0O/dzCDq4w5G2+KRLUBdqKm0eAKA=";
```

```suggestion
    hash = "sha256-CLcr+X/b0moVEBV0O/dzCDq4w5G2+KRLUBdqKm0eAKA=";
```
hash is the more modern thing to use.